new/LIT7158PPAynaTe.xml

```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7158PPAynaTe" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against the evil eye and for the drowning of demons, containing ʾasmāt and the names of the days of the week</title>
+                <title xml:lang="gez" xml:id="t2">ጸሎት፡ በእንተ፡ ዓይነ፡ ጥላ፡ ወመስጥመ፡ አጋንንት፡ (...)</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Ṣalot baʾǝnta ʿayna ṭǝlā wa-masṭǝma ʾagānǝnt (...)</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0.</licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p>The text is contained in the manuscript <ref type="mss" corresp="RDW009"/>.</p>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
```

Comment:
You can add an abstract similar to the one in LIT7074AynatTsella and point to the similarity to LIT7074AynatTsella and other magic prayers in this abstract and in relations:

```

<p>Protective prayer against the evil action described as <foreign xml:lang="gez-tr">ʾaynat</foreign> ("evil eye") of 
                    <foreign xml:lang="gez-tr">ṣǝlā</foreign>, which may be an abbreviated form of 
                    <foreign xml:lang="gez-tr">ṣǝlā wagi</foreign> (literally: "piercer of the shadow"), who was described as a sorcerer or as a 
                    demon by <bibl><ptr target="bm:Worrell1914Zauberwesen"/><citedRange unit="page">136</citedRange></bibl> and 
                    <bibl><ptr target="bm:Strelcyn1972Wellcome"/><citedRange unit="page">52</citedRange></bibl>.</p>
```